<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GloSCClass</class>
 <widget class="QMainWindow" name="GloSCClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>386</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>386</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>386</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GloSC</string>
  </property>
  <property name="windowIcon">
   <iconset resource="GloSC.qrc">
    <normaloff>:/GloSC/Resources/GloSC_Icon.png</normaloff>:/GloSC/Resources/GloSC_Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Instances</string>
    </property>
    <widget class="QListWidget" name="lwInstances">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>21</y>
       <width>261</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbAddToSteam">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>330</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Add all to Steam</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbCreateNew">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="configBox">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>10</y>
      <width>391</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>29</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leName">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>29</y>
       <width>331</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbOverlay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>371</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable overlay</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbControllers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>371</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable virtual controllers</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbUseDesktop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Use Desktop-Config
Disables hooking, per application bindings won't work</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbHookSteam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Hook Steam
Enforces bindings reliably by hooking into Steam</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbLaunchGame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>371</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Launch game</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>231</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Path: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lePath">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>231</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSearchPath">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>230</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbUWP">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>230</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>UWP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>255</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Arguments: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leArguments">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>255</y>
       <width>310</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbCloseWhenDone">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Close shortcut when launched Application closes
(Disable for Uplay-games!)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSave">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 8pt &quot;Arial Black&quot;;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbDelete">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GloSC.qrc"/>
 </resources>
 <connections/>
</ui>
